src/controllers/UserController.js

```diff
@@ -40,11 +40,20 @@ export default class UserController {
    * @return {object} return all users in database
    */
   static async getAllUser(req, res) {
-    const role = 'normal';
-    const findAll = await User.getAllUser({ role });
+    const role = req.user.role === 'normal' ? 'normal' : null;
```

Comment:
Is this how we are supposed to implement permissions?

Reply:
I implemented this because there is a colleague who is working on getting all users, to make the method reusable, I implemented like that.

Reply:
if it shouldn't be part of this, I would advise you remove this because I have issues with this impelementation

Reply:
Do u mean to remove it and keep the previous?